docs/standard/io/media/pipelines/code/DoNotUse.cs

```diff
@@ -0,0 +1,211 @@
+﻿using System;
```

Comment:
code should go to dotnet/samples

Reply:
I'll move them to snippets.

Reply:
@mairaw  can you move the snippets over and fix the snippet ref in the doc? I have no idea how to do that.

Reply:
Yes, I can do that.